src/Resources/Resources/ChangeLog.md

```diff
@@ -26,6 +26,7 @@
 * Add support of policy exemption resource type
 * Fixed what-if functionality when using `-QueryString` parameter.
 * Normalized `-QueryString` starting with "?" for scenarios involving dynamic parameters.
+* Add PrincialType to role assignment requests
```

Comment:
Please merge master branch into your branch. Changelog msgs should be added under the top of ##upcoming release section. You should detect one conflict if you do so.

Reply:
changed